recipe/meta.yaml

```diff
@@ -31,8 +31,11 @@ test:
     - numpy >=1.6.0
   imports:
     - pytools
+  source_files:
+    - pytools
```

Comment:
Is this needed?

Reply:
Without it, I get 

```
+ pip check
No broken requirements found.
+ /Users/mdiener/Work/pytools-feedstock/miniforge3/conda-bld/pytools_1711034899242/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/bin/python -m pytest pytools
============================= test session starts ==============================
platform darwin -- Python 3.12.2, pytest-8.1.1, pluggy-1.4.0
rootdir: $SRC_DIR
collected 0 items

============================ no tests ran in 0.00s =============================
ERROR: file or directory not found: pytools
```

Reply:
@matthiasdiener when the tests are part of the package we can use `pyargs` to find them. The last commit should fix this.

Reply:
Thanks @ocepaf! It seems that with this change, we also don't need `source_files` anymore ([83cec2f](https://github.com/conda-forge/pytools-feedstock/pull/65/commits/83cec2f14bd0ad8453213c96a2c4be6ccc7c1abb)).